internal/gps/strip_vendor_windows.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 func stripVendor(path string, info os.FileInfo, err error) error {
+	if err != nil && err != filepath.SkipDir {
```

Comment:
Have you actually seen `filepath.SkipDir` here? Wouldn't that be a sever std lib bug?